galasa-managers-parent/galasa-managers-cicsts-parent/dev.galasa.cicsts.resource.manager/src/main/java/dev/galasa/cicsts/resource/internal/CicsBundleImpl.java

```diff
@@ -406,11 +340,11 @@ public boolean isEnabled() throws CicsBundleResourceException {
 	public boolean disable() throws CicsBundleResourceException {
         try {
             if (!resourceInstalled()) {
-                throw new CicsJvmserverResourceException(RESOURCE_TYPE_BUNDLE + " " + getName() + " does not exist");
+                throw new CicsJvmserverResourceException(RESOURCE_TYPE_BUNDLE + " " + getDefinitionName() + " does not exist");
             }
-            this.cicsRegion.cemt().setResource(this.cicsTerminal, RESOURCE_TYPE_BUNDLE, getName(), "DISABLED");
+            this.cicsRegion.cemt().setResource(this.cicsTerminal, RESOURCE_TYPE_BUNDLE, getDefinitionName(), "DISABLED");
         } catch (CicstsManagerException e) {
-            throw new CicsBundleResourceException("Problem disabling " + RESOURCE_TYPE_BUNDLE + " " + getName(), e);
+            throw new CicsBundleResourceException("Problem disabling " + RESOURCE_TYPE_BUNDLE + " " + getDefinitionName(), e);

```

Comment:
Is it ever possible that getDefinitionName returns null ? In which case would this cause a NullPtrException ? Better to use one of the formatting APIs like MessageFormat.format(template,parameter...) which copes with nulls as substitution strings ?

Reply:
Not got to try it as there's no jvmserver support for my test bundles.